src/inspector/models/notation/tremolos/tremolosettingsmodel.cpp

```diff
@@ -70,6 +70,11 @@ void TremoloSettingsModel::resetProperties()
     m_direction->resetToDefault();
 }
 
+void TremoloSettingsModel::onNotationChanged(const PropertyIdSet&, const StyleIdSet&)
+{
+    loadProperties();
+}
```

Comment:
NB at least InspectorModelWithVoiceAndPositionOptions has exactly the same implementation, other models tend to check whether the changed property set contains the relevant ID, but it strikes me as premature optimisation...

Reply:
And it's not perfectly done in that model either. You should check if PropertyIdSet contains the required properties

Reply:
Ok so can I propose as a compromise:

a) we add "loadProperties( const PropertyIdSet&, const StyleIdSet&) to AbstractInspectorModel, and the default implementation of onNotationChanged is to call this
b) all models should only have a single loadProperties( ) function, whereby the assumption is that if the sets are empty, then all properties should be loaded.

Because the current duplication of code seems to be highly likely to lead to unexpected behaviour in the future if one version of loadProperties( ) is updated differently from the other.

Reply:
Let's avoid any refactoring within the scope of this PR and only change the implementation of TremoloSettingsModel::onNotationChanged

Reply:
But then you're asking me to continue the pattern of code duplication in the other models that we really should be trying to avoid. If you'd prefer to do the refactoring yourself then ok, as that will fix this issue and prevent similar future issues.

Reply:
If you can advise how we can get this closed without the code duplication issue that exists in the other property model classes I'd like to get this one approved and merged.

Reply:
@wizofaus I would suggest to look at `AppearancePreferencesModel::loadProperties`. That implementation is perhaps not perfect either, but probably the best we can get without a lot of refactoring.

Reply:
Seems a reasonable compromise, done.